includes/ucf-today-functions.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Functions related to retrieval of curated news
+ * articles or fallback content from UCF Today
+ */
+namespace GMUCF\Theme\Includes\UCFToday;
+use GMUCF\Theme\Includes\Utilities;
+
+
+/**
+ * Logic for determining featured stories content
+ *
+ * @return array
+ * @author Chris Conover
+ **/
+function get_featured_stories_details( $limit = 2 ) {
+	$stories = array();
+	$main_site_stories_rss_url = get_option( 'main_site_stories_url' );
+	$main_site_stories_timeout = get_option( 'main_site_stories_timeout' );
+
+	$rss = Utilities\custom_fetch_feed( $main_site_stories_rss_url . '?thumb=gmucf_featured_story', $main_site_stories_timeout );
+
+	if( !is_wp_error( $rss ) ) {
+		$rss_items = $rss->get_items( 0, $rss->get_item_quantity( 15 ) );
+
+		$count = 0;
+		$top_story_id = get_transient( 'top_story_id' );
+		foreach( $rss_items as $rss_item ) {
+			if( $count == $limit ) break;
```

Comment:
Make this a strict equality.

```suggestion
			if( $count === $limit ) break;
```

Reply:
I apparently missed a bunch of these throughout the theme.  This particular line has been fixed, but I've opened a new issue to address the rest: https://github.com/UCF/UCF-GMUCF-Theme/issues/70